packages/indexer-common/src/indexer-management/__tests__/allocations.test.ts

```diff
@@ -99,7 +100,9 @@ const teardownAll = async () => {
 
 describe('Allocation Manager', () => {
   beforeAll(setup)
-  beforeEach(setupEach)
+  beforeEach(() => {
+    return setupEach()
+  })
```

Comment:
I think the new form is the same as the previous one.
```suggestion
  beforeEach(setupEach)
```

Reply:
ah, yes, sorry, I did this change as I was playing around and missed it in the cleanup!